webpages/settings/components/addon-setting.html

```diff
@@ -396,4 +406,11 @@
     border-top-left-radius: 4px;
     border-top-right-radius: 0;
   }
+  .hide {
+    display: none;
+  }
+  .disableEditing input[type="text"] {
+    pointer-events: none;
+    border: none;
```

Comment:
I think disabling text inputs and adding or removing table items should be separate